src/main/java/edu/lwtech/csd297/teachersfirst/actions/DeleteOpeningAction.java

```diff
@@ -43,7 +43,7 @@ public void RunAction() {
 		//logger.info(DataManager.getOpeningDAO().size() + " records total");
 		logger.debug("Deleted opening ID: [{}]", openingIdInt);
 		
-		this.SendPostReply("/openings", "", "Opening %5B" + openingIdString + "%5D deleted!");
+		this.SendPostReply("/openings", "", "Opening %5B" + openingIdString + "%5D, deleted!");
```

Comment:
This seems... from out of nowhere?

Reply:
Yeah, I don't want to make a commit just to undo that, not sure why it got put in though.